living_hinge.py

```diff
@@ -140,9 +142,9 @@ def generate(self, swatch):
         }
         path_command = ""
         y = self.y
-        while y < self.y + self.height:
+        for _ in range (y_patterns):
```

Comment:
I think y_patterns needs to be wrapped in either round() or int() to work when multiple objects are selected.